src/boosting/gbdt.cpp

```diff
@@ -581,17 +580,20 @@ void GBDT::PredictContrib(const double* features, double* output, const Predicti
     for (int k = 0; k < num_tree_per_iteration_; ++k) {
       models_[i * num_tree_per_iteration_ + k]->PredictContrib(features, num_features, output + k*(num_features + 1));
     }
-    // check early stopping
-    ++early_stop_round_counter;
```

Comment:
@slundberg checking if this is ok to remove, I don't think early stopping is ever used/applies here

Reply:
if you mean that early stopping is not exposed in the external API, then I would think we should expose it rather than remove support. If I train a model with 10k trees and then decide to use just the first 3k for prediction then I think we want the same ability to explain the model that only uses the first 3k trees.

Reply:
@imatiach-msft clarified that this is for the number of samples not ntrees so I agree it can go.